variables.tf

```diff
@@ -77,6 +77,23 @@ variable "create_cos_bucket" {
   default     = true
 }
 
+variable "cross_region_location_enabled" {
```

Comment:
Do we really need this extra variable? I think we can remove it and just have validation logic that would fail if values for var.region and var.cross_region_location were both passed